src/Microphysics1M.jl

```diff
@@ -215,7 +217,10 @@ function terminal_velocity(
         (; ϕ, κ) = aspr
 
         # eq 20 from Chen 2022
-        fall_w = ϕ^κ * sum(CO.Chen2022_exponential_pdf.(aiu, bi, ciu, λ, 3))
+        for it in range(start=1, step=1, length=2)
+            fall_w += ϕ^κ * CO.Chen2022_exponential_pdf(aiu[it], bi[it], ciu[it], λ, 3)
+        end
```

Comment:
```suggestion
        fall_w = sum((1,2,3); init=FT(0)) do i
            ϕ^κ * CO.Chen2022_exponential_pdf(aiu[i], bi[i], ciu[i], λ, 3)
        end
```
This is probably the simplest gpu-friendly pattern that I would recommend applying here. If the tuple was larger, I might say do
`ntuple(i->i, N)` where `N` is the tuple length.